assets/dev/scss/frontend/_global.scss

```diff
@@ -101,6 +106,117 @@
 	}
 }
 
+
```

Comment:
You should export the exist `ui-resizable` rules into a new file and import it in editor & editor preview, instead of duplicating the code.